hoomd/mpcd/DryingDropletStreamingMethod.cc

```diff
@@ -121,5 +127,7 @@ void mpcd::detail::export_DryingDropletStreamingMethod(pybind11::module& m)
     {
     namespace py = pybind11;
     py::class_<mpcd::DryingDropletStreamingMethod, mpcd::ConfinedStreamingMethod<mpcd::detail::SphereGeometry>, std::shared_ptr<mpcd::DryingDropletStreamingMethod>>(m, "DryingDropletStreamingMethod")
-        .def(py::init<std::shared_ptr<mpcd::SystemData>, unsigned int, unsigned int, int, std::shared_ptr<::Variant>, boundary, Scalar, unsigned int>());
+        .def(py::init<std::shared_ptr<mpcd::SystemData>, unsigned int, unsigned int, int, std::shared_ptr<::Variant>, boundary, Scalar, unsigned int>())
+        .def("getFiller", &mpcd::DryingDropletStreamingMethod::getFiller)
+        .def("setFiller", &mpcd::DryingDropletStreamingMethod::setFiller);
```

Comment:
pybind11 properties are a nicer interface for the get-set pattern:
```suggestion
        .def_property("filler", &mpcd::DryingDropletStreamingMethod::getFiller, &mpcd::DryingDropletStreamingMethod::setFiller);
```
See the python file for how to use this.